taxi/forms.py

```diff
@@ -0,0 +1,46 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverForm(UserCreationForm):
+    class Meta(UserCreationForm):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("first_name", "last_name", "license_number",)
+

```

Comment:
Add check for validity of the `license_number` field when creating driver. This is a checklist requirement.